### DISCUSSION TITLE
165

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#115] [General]
# 潮流周刊第165期 - 年度榜单

封面图来源于上海浦东美术馆曾梵志展览的一张布面油画，很神奇，人眼看看不出人物画像，但是用手机拍摄可以看到人物的样子。本期周刊将 2023 年 50 多期内容选择出一批我觉得不错的的学习/视听/工具/素材的网站汇总分析给大家。

https://weekly.tw93.fun/posts/165-%E5%B9%B4%E5%BA%A6%E6%A6%9C%E5%8D%95

**Comment by i-vox** [comment]
这一期质量很高，赞👍🏻
